Cargo.toml

```diff
@@ -34,6 +34,7 @@ digest = { version = "0.9", features = ["dev"] }
 default = ["aead", "alloc", "digest", "signature"]
 alloc = ["aead/alloc"]
 signature = ["ecdsa", "ed25519", "p256", "p384"]
+std = ["digest/std"]
```

Comment:
It would be good to activate `signature/std` as well

Reply:
Makes sense - I've added a commit that enables the std feature on each of the signature-releated crates (can't do signature/std directly since signature isn't included directly in this crate)